randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -756,12 +785,23 @@ Extra - asset_id: collision_camera_018
   * Pickup 1; Category? Minor
   * Extra - actor_name: LE_Item_002
   * Extra - actor_type: item_senergytank
+  > Room Center
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Phase Drift or Use Spider Ball
+              All of the following:
+                  # Phase Drift Skip
+                  Lay Bomb
+                  Any of the following:
+                      Phase Drift Skip (Expert)
+                      Space Jump and Phase Drift Skip (Advanced)
   > Below Right Crumble Blocks
       Morph Ball
 
 > Door to Landing Site; Heals? False
   * Layers: default
-  * Spazer Beam Door to Landing Site/Door to Surface Stash
+  * Spazer Beam Door to Landing Site/Door to Surface Stash; Dock Lock Rando incompatible with: Access Open
```

Comment:
Still don't think we need to have this exclusion since we'll likely never have removed doors.

Reply:
Better safe than sorry IMO.